src/SprykerEco/Yves/Braintree/Theme/default/components/molecules/pay-pal/pay-pal.ts

```diff
@@ -0,0 +1,54 @@
+import PaymentForm from '../payment-form/payment-form';
+
+export default class PayPal extends PaymentForm {
+    form: HTMLFormElement;
+    braintreePayPalMethod: HTMLElement;
+
+    readonly braintreePayPal: string = 'braintreePayPal';
+    readonly PayPalAccount: string = 'PayPalAccount';
+
+    protected readyCallback(): void {
+        this.form = <HTMLFormElement>document.getElementById(`${this.formId}`);
+        this.braintreePayPalMethod = <HTMLElement>this.form.querySelector(`.${this.jsName}__method`);
+
+        super.readyCallback();
+    }
+
+    protected errorHandler(error: any) {
```

Comment:
Please create this method implementation inside PaymentForm. The only difference - check with current payment method.